Deceive/MainController.cs

```diff
@@ -31,17 +33,34 @@ public MainController()
             trayIcon.ShowBalloonTip(5000);
             LoadStatus();
             SetupMenuItems();
+
+            //Update status when LCU launches
+            //Try every 5 seconds until LCU is ready to accept our status
+            initLCUStatusTimer.Tick += new EventHandler(InitLCUStatus);
+            initLCUStatusTimer.Interval = 5000;
+            initLCUStatusTimer.Start();
+        }
+
+        private void InitLCUStatus(object sender, EventArgs e)
+        {
+            try
+            {
+                Utils.SendStatusToLCU(status);
+                //stop Timer when changing LCU status finally worked and we get no WebException
+                initLCUStatusTimer.Dispose();
+            }
+            catch { /* LCU is not ready yet. */ }
         }
 
         private void SetupMenuItems()
         {
-            var aboutMenuItem = new MenuItem("Deceive v1.4.1");
+            var aboutMenuItem = new MenuItem("Deceive v1.5.0");
             aboutMenuItem.Enabled = false;
 
             var enabledMenuItem = new MenuItem("Enabled", (a, e) =>
             {
                 enabled = !enabled;
-                UpdateStatus(enabled ? status : "online");
+                UpdateStatus(enabled ? status : "chat");
```

Comment:
Good catch. Despite being fully aware that it's `chat` and not `online`, I guess it just slipped through .